teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>We skip the protonation, since openmm can do this later as well. However, I am not sure if it will really consider the small molecule during protonation of the protein. Please double check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>